README.md

```diff
@@ -40,142 +38,124 @@
-- [ProductFeed](https://productfeed.app) - Retain users and keep them updated.
-- [Changes](https://changes.blue) - Keep users in the loop about product updates with a modern changelog that grows with you.
+- [Changefeed](https://changefeed.app/) - Set up a changelog page or Embed your changelog in your website.
+- [Canny's Changelog](https://canny.io/features/changelog) - Combines changelog with product roadmaps.
+- [ProductFlare's Changelog](https://www.productflare.com/changelog) - Changelog widget around a product team task management system.
+- [Changes](https://changes.blue) - Changelog building kit.
 
 ## Telecommunication
-- [Vonage](https://www.vonage.com/) - Flexible and scalable voice, messaging, video and data capabilities across Unified Communications, Contact Centers and Communications APIs.
-
-## Privacy
-- [Skyflow](https://www.skyflow.com/) - API for privacy.
+- [Vonage](https://www.vonage.com/) - Voice, messaging, video and data capabilities across Unified Communications, Contact Centers and Communications APIs.
 
 ## Social Media
-- [AddThis](https://www.addthis.com/) - Turn visitors into engaged customers.
-- [ShareThis](https://sharethis.com/) - Grow audience and keep them engaged.
-- [AddToAny](https://www.addtoany.com/) - Universal sharing.
-- [WarefarePlugins](https://warfareplugins.com/) - Social sharing for WordPress.
-
-## Sales Funnel
-- [Click Funnels](https://www.clickfunnels.com/) - Create sales funnels that convert visitors into leads and then customers.
-- [CartFlows](https://cartflows.com/) - Get more leads, increase conversions, & maximize profits.
+- [AddThis](https://www.addthis.com/) - Social sharing buttons, available on AMPs.
+- [ShareThis](https://sharethis.com/) - Social sharing buttons.
+- [AddToAny](https://www.addtoany.com/) - Sharing buttons for ~100 different target services.
 
 ## Search
-- [Algolia](https://www.algolia.com/) - Make every search releavnt, immersive, optimized, engaing, intelligent, personalized, business-driven, etc.
+- [Algolia](https://www.algolia.com/) - Search for your application.
 
 ## Feature flags
-- [Optimizely's Rollouts](https://www.optimizely.com/rollouts) - Ship faster and prove value with less risk.
-- [LaunchDarkly](https://launchdarkly.com/) - Manage features at any scale.
-- [Rollout](https://rollout.io/) - Progressively deliver features to users with confidence, in real-time.
-- [Split](https://www.split.io/) - Confidently release features as fast as you develop them.
+- [Optimizely's Rollouts](https://www.optimizely.com/rollouts) - Feature flags combined with A/B testing.
+- [LaunchDarkly](https://launchdarkly.com/) - Feature management with multiple integrations in CI servers and ticketing systems.
+- [Split](https://www.split.io/) - Another feature management with multiple integrations in CI servers and ticketing systems.
```

Comment:
```suggestion
- [Split](https://www.split.io/) - Feature management with multiple integrations in CI servers and ticketing systems.
```

Reply:
That's the same description as the item above which looks like a copy-paste error. That's why I added "another" to make clear that it isn't.